test/alphabet/aminoacid_test.cpp

```diff
@@ -0,0 +1,178 @@
+// ==========================================================================
+//
+// Copyright (c) 2006-2017, Knut Reinert, FU Berlin
+// Copyright (c) 2016-2017, Knut Reinert & MPI Molekulare Genetik
+// All rights reserved.
+//
+// Redistribution and use in source and binary forms, with or without
+// modification, are permitted provided that the following conditions are met:
+//
+//     * Redistributions of source code must retain the above copyright
+//       notice, this list of conditions and the following disclaimer.
+//     * Redistributions in binary form must reproduce the above copyright
+//       notice, this list of conditions and the following disclaimer in the
+//       documentation and/or other materials provided with the distribution.
+//     * Neither the name of Knut Reinert or the FU Berlin nor the names of
+//       its contributors may be used to endorse or promote products derived
+//       from this software without specific prior written permission.
+//
+// THIS SOFTWARE IS PROVIDED BY THE COPYRIGHT HOLDERS AND CONTRIBUTORS "AS IS"
+// AND ANY EXPRESS OR IMPLIED WARRANTIES, INCLUDING, BUT NOT LIMITED TO, THE
+// IMPLIED WARRANTIES OF MERCHANTABILITY AND FITNESS FOR A PARTICULAR PURPOSE
+// ARE DISCLAIMED. IN NO EVENT SHALL KNUT REINERT OR THE FU BERLIN BE LIABLE
+// FOR ANY DIRECT, INDIRECT, INCIDENTAL, SPECIAL, EXEMPLARY, OR CONSEQUENTIAL
+// DAMAGES (INCLUDING, BUT NOT LIMITED TO, PROCUREMENT OF SUBSTITUTE GOODS OR
+// SERVICES; LOSS OF USE, DATA, OR PROFITS; OR BUSINESS INTERRUPTION) HOWEVER
+// CAUSED AND ON ANY THEORY OF LIABILITY, WHETHER IN CONTRACT, STRICT
+// LIABILITY, OR TORT (INCLUDING NEGLIGENCE OR OTHERWISE) ARISING IN ANY WAY
+// OUT OF THE USE OF THIS SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH
+// DAMAGE.
+//
+// ==========================================================================
+
+#include <sstream>
+
+#include <gtest/gtest.h>
+
+#include <range/v3/view/zip.hpp>
+
+#include <seqan3/alphabet/aminoacid/all.hpp>
+
+using namespace seqan3;
+using namespace seqan3::literal;
+
+template <typename T>
+class aminoacid : public ::testing::Test
+{};
+
+// add all alphabets from the aminoacid sub module here
+using aminoacid_types  = ::testing::Types<aa27>;
+using aminoacid_types2 =       meta::list<aa27>; // needed for some tests
+
+TYPED_TEST_CASE(aminoacid, aminoacid_types);
+
+TYPED_TEST(aminoacid, static_members)
+{
+    TypeParam::A;
+    TypeParam::B;
+    TypeParam::C;
+    TypeParam::D;
+    TypeParam::E;
+    TypeParam::F;
+    TypeParam::G;
+    TypeParam::H;
+    TypeParam::I;
+    TypeParam::J;
+    TypeParam::K;
+    TypeParam::L;
+    TypeParam::M;
+    TypeParam::N;
+    TypeParam::O;
+    TypeParam::P;
+    TypeParam::Q;
+    TypeParam::R;
+    TypeParam::S;
+    TypeParam::T;
+    TypeParam::U;
+    TypeParam::V;
+    TypeParam::W;
+    TypeParam::X;
+    TypeParam::Y;
+    TypeParam::Z;
+    TypeParam::TERMINATOR;
+    TypeParam::UNKNOWN;
+}
```

Comment:
This function will just produce warnings (as soon as they are turned on via #63 ) so I would propose to just remove it. We can think of something similar later!